### PR TITLE
Team - Remove jomey

### PR DESCRIPTION
**Pull request by jomey**

### CODE REVIEW

**Comment by jomey** [comment]
#39 addresses the failing build from above

File: book/team.yaml [hunk 1]
```diff
@@ -143,29 +143,6 @@ people:
...
-      - Python
-      - git, GDAL, anything in the Terminal really ...
-      - Modeling
-    social:
-      - icon: github
-        icon_pack: fab
-        link: https://github.com/jomey
-    user_groups:
-      - Technology Specialist
-  # =============
```
Comment (scottyhq) [inline book/team.yaml:168]: i'd be happy to leave you on the webpage for all your contributions to the website and overall design @jomey, but also understand if you just prefer to sit this one out!

Comment (jomey) [reply book/team.yaml:168]: I won't attend the event in any capacity and all other contributions are recorded with the website GH repo. That's enough for me :smiley:

**Review by aaarendt — APPROVED** [review]

**Comment by aaarendt** [comment]
@jomey just confirming we should merge and override the QC error, since this is fixed elsewhere?

**Comment by jomey** [comment]
> @jomey just confirming we should merge and override the QC error, since this is fixed elsewhere?

Correct, the QC error is fixed per #39

**Pull request merged**